modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/data_renderers/TooltipPriceRenderer.js

```diff
@@ -58,17 +58,15 @@ function TooltipPriceRenderer(props) {
 		<>
 			{props.value.final.value}
 			{props.value.details && (
-				<ClayTooltipProvider
-					contentRenderer={() => <TooltipTable {...props} />}
-					delay={0}
+				<span
+					className="tooltip-provider"
+					title={ReactDOMServer.renderToString(
+						<TooltipTable {...props} />
+					)}
+					data-tooltip-delay="0"
 				>
-					<span
-						className="tooltip-provider"
-						title={Liferay.Language.get('price-summary')}
-					>
-						<ClayIcon symbol="info-circle" />
-					</span>
-				</ClayTooltipProvider>
+					<ClayIcon symbol="info-circle" />
+				</span>
```

Comment:
How are you getting the `{Liferay.Language.get('price-summary')}` to this new tooltip? It seems that with this replacement you've lost it, or it's getting propagated to it in some other way?